testnets/xiontestnet/chain.json

```diff
@@ -32,26 +33,15 @@
   },
   "codebase": {
     "git_repo": "https://github.com/burnt-labs/xion",
-    "recommended_version": "v0.3.9",
```

Comment:
Although the `versions` array is populated nicely, why has the version info at a higher level in this codebase object been removed? This would be best to keep (reflective of v9.0.0)

Reply:
We have reversioned our repo prior to mainnet launch to adhere to semantic versioning rules, version v0.3.9==v6.0.0
see: https://github.com/burnt-labs/xion/releases/tag/v6.0.0